### ISSUE TITLE
avoid objects is broken

### ISSUE THREAD

**Issue by autobot00** [EasyFarm/EasyFarm#349]
back in earlier versions avoid objects would "re-position" you and then go engage the mob.  Now it just repeatedly rotates you around and moves you and never engages anything even if you are re-positioned and have a free path to the mob.  it never attempts to do anything other than repeatedly re-position.

TLDR, its broked.

**Comment by Mykezero** [comment]
Yes, I don't think it ever worked quite well. It definitely needs some love and a bit of an overhaul.

**Comment by Mykezero** [comment]
And thanks for opening the issue! I kind of forgot about this feature ^^

**Comment by autobot00** [comment]
it actually did work pretty good a few versions ago.  i could actually run it in qufim where its got those stupid raised portions and it actually did good going around them.  If you changed the coding from that maybe put it back to where it was.

**Comment by Mercurian** [comment]
In earlier versions it would just clip. That's why it works so well lol. Problem was it would do it without actually getting stuck and is pretty noticeable. 

if you're talking about the early 1.0.110~ releases
